src/deterministic_vi/elbo.jl

```diff
@@ -235,11 +235,12 @@ function accum_star_pos!{NumType <: Number}(
                     x::Vector{Float64},
                     wcs_jacobian::Array{Float64, 2},
                     calculate_derivs::Bool)
+
     eval_bvn_pdf!(elbo_vars.bvn_derivs, bmc, x)
 
-    # TODO: Also make a version that doesn't calculate any derivatives
-    # if the object isn't in active_sources.
-    get_bvn_derivs!(elbo_vars.bvn_derivs, bmc, true, false)
+    if elbo_vars.calculate_derivs && calculate_derivs
```

Comment:
As is, the derivative isn't necessarily calculated when the local  `calculate_derivs` is true. So it's a bad or at least misleading variable name. I like `is_active_source`, because it explains why you're overriding `elbo_vars.calculate_derivs`. Other names could work too.